configure.ac

```diff
@@ -810,6 +791,44 @@ AS_IF([test x$with_lustre != xno],[
        MOTR_LINUX_SENDFILE
 ])
 
+#
+# Checking Intel ISA library --------------------------------------------------------- {{{1
+#
+AC_ARG_ENABLE([isal],
+        [AS_HELP_STRING([--disable-isal],
+                        [Disable use Intel ISA. Instead use Galois library.])],
+        [],[enable_isal=yes]
+)
+
+AM_CONDITIONAL([HAVE_ISAL],
+               [test "x$enable_isal" = xyes])
+
+
+AS_IF([test x$enable_isal = xyes],
+      [
+              AC_CHECK_HEADERS([isa-l.h],
+              [
+                      # check for isa library
+                      MOTR_SEARCH_LIBS(
+                              [ec_encode_data], [c isal], [ISAL_LIBS],
+                              [ec_encode_data cannot be found! \
+                               Try to install Intel ISA library package.]
+                      )
+                      AC_SUBST([ISAL_LIBS])
+
+                      # define MACROs
+                      AC_DEFINE([HAVE_ISAL])
+                      # AM_CONDITIONAL([HAVE_ISAL], [true])
+                      AC_MSG_NOTICE([INTEL ISA-L Headers Available])
+              ],
+              [
+                      AM_CONDITIONAL([HAVE_ISAL], [false])
+                      AC_MSG_NOTICE([No INTEL ISA-L Headers])
```

Comment:
I propose to replace this line with:
AC_MSG_ERROR([No INTEL ISA-L Headers])

Reasons:
The default mode (no isa related arguments provided or --enable-isal is provided) is "enable-isal".
So, the ISA-L header must be present. Otherwise, we should report error. We will not fall back to Galois.

If "--disable-isal" is provided, we will not check ISA-L header files, and just define HAVE_ISAL as false.

Reply:
But, after some testing and more thinking, maybe you are right.
We can fall back to Galois if no Intel ISA is installed.